### ISSUE TITLE
Load Testing

### ISSUE THREAD

**Issue by MarsRomer** [bcgov/CONN-CCBC-portal#916]
Load test concurrent multiple file uploads of large sizes <100mb 

K6
Testing:
200 people concurrently uploading large files <100mb

**Comment by AntBush** [comment]
[Test Results csv](https://app.zenhub.com/files/476885245/f9f78c2f-12fc-4aaa-ab62-599a9d445f5b/download)

**Comment by marcellmueller** [comment]
Yesterday Anthony and I did the load testing on our development environment. Unfortunately the upload load tests are somewhat constrained by our computers memory and our network speed but we ran the tests for about 45 minutes and uploaded nearly 400 files to our S3 bucket.

My pc has less RAM so I ran the small uploads test which consisted of file sizes that will likely be the majority of our files. It peaked at 30 virtual users uploading 100KB and 1MB files.

While I was running that Anthony concurrently ran the large upload test which peaked at 10 virtual users uploading 10MB, 50MB and 100MB files.

Overall the tests went well and our deployment handled it admirably. We could definitely try and push it harder in the future either by running tests from a virtual machine on AWS or something or getting more devs to run them concurrently. 

Action items from this are to enable horizontal pod autoscaling and possibly request more CPU in our namespaces. This would allow us to go from running our application on 3 pods to likely 6 during periods of heavy load, then back to 3 for when things quieted down.
